src/index.css

```diff
@@ -1,3 +1,146 @@
-* {
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;700&display=swap');
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
   margin: 0;
+  padding: 0;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+  height: 100%;
+  width: 100vw;
+}
+
+#root {
+  width: 100%;
+  height: 100%;
+}
```

Comment:
I would like to note that the width and height properties of an element are by default set to 100%. That means there is no need to add them unless you want to modify them. I kindly suggest that you may remove them if you don't need to adjust them.